compare_mt/reporters.py

```diff
@@ -114,45 +120,41 @@ def print_header(self, header):
   def generate_report(self, output_fig_file=None, output_fig_format=None, output_directory=None):
     self.print()
 
-    if output_fig_file is not None:
-      self.plot(output_directory, output_fig_file, output_fig_format)
-
 class ScoreReport(Report):
-  def __init__(self, scorer_name, scores, strs, 
+  def __init__(self, scorer, scores, strs, 
                wins=None, sys_stats=None, compare_directions=None):
-    self.scorer_name = scorer_name 
+    self.scorer = scorer 
     self.scores = scores
     self.strs = strs
     self.wins = wins
     self.sys_stats = sys_stats 
-    self.compare_directions = compare_directions
-    self.output_fig_file = None
+    self.wins = wins
```

Comment:
Should this be `self.compare_directions=compare_directions`?

Reply:
Thanks! Fixed.